pagopa/io-app
# chore(IT Wallet): [SIW-907] Add StrongBox Mixpanel tracking for Lollipop keys

## Short description
This PR adds a Mixpanel tracking event which indicates wether or not the Lollipop generated keys are backed by StrongBox on Android.
The event is tracked during startup on already generated Lollipop keys, otherwise we woulnd't have this information until 365 days for LV sessions or 30 days for standard sessions.

## List of changes proposed in this pull request
- Updates `io-react-native-crypto` to version `0.3.0` which implements the `isKeyStrongBoxBacked` function;
- Adds a `handleIsKeyStrongboxBacked` method which tracks the success or failure information by calling `isKeyStrongBoxBacked` only on Android and skips iOS entirely.
- Adds a call to `handleIsKeyStrongboxBacked` in the startup saga.

## How to test
Test a login flow on Android devices which support TEE and StrongBox. Check on Mixpanel if the event `LOLLIPOP_IS_KEY_STRONGBOX_BACKED_SUCCESS` is properly tracked with the `isStrongboxBacked` property which should be `false` in case of TEE and `true` in case of StrongBox.
